### PR TITLE
Rename the globals to shorter names.

### PR DESCRIPTION
**Pull request by brendandahl**
pdfjsDistBuildPdf=pdfjsLib
pdfjsDistWebPdfViewer=pdfjsViewer
pdfjsDistBuildPdfWorker=pdfjsWorker

See #9545

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -261,10 +258,11 @@ function createBundle(defines) {
     libraryTarget: 'umd',
     umdNamedDefine: true,
   });
+
   var workerOutput = gulp.src('./src/pdf.worker.js')
     .pipe(webpack2Stream(workerFileConfig))
     .pipe(replaceWebpackRequire())
-    .pipe(replaceJSRootName(workerAMDName));
+    .pipe(replaceJSRootName(workerAMDName, 'pdfjsWorker'));
```
Comment (Snuffleupagus) [inline gulpfile.js]: The rest of the code uses `pdfjsLibWorker`, so this line (or the rest of the patch) needs to be fixed :-)

**Review by yurydelendik — APPROVED** [review]

**Pull request merged**